vispy/gloo/glir.py

```diff
@@ -18,6 +18,31 @@
 from ..ext.six import string_types
 from ..util import logger
 
+# TODO: expose these via an extension space in .gl?
+_internalformats = [
+    gl.Enum('GL_R', 8194), 
+    gl.Enum('GL_R8', 33321), 
```

Comment:
I think specifying the enums here is alright for now, it's easily changed if necessary when we come up with a final solution.